docs/Writerside/topics/resources/data_format.md

```diff
@@ -0,0 +1,106 @@
+# Data Format
+
+During a recording, FreeMoCap outputs numerous files. Some are formatted for easy use in other programs, while some are primarily used as intermediary steps during processing. FreeMoCap keeps all the files created during processing, which allows you to restart processing from any part of the pipeline, and gives you fine-grained control of what output data you use. All the files from a recording session are saved to the recording folder (see the [Freemocap Data Folder Guide](freemocap_data_folder_guide.md)).
+
+You can see the layout of a FreeMoCap recording session below:
+```
+freemocap_sample_data
+├── annotated_videos
+├── freemocap_sample_data.blend
+├── freemocap_sample_data.ipynb
+├── freemocap_sample_data_by_frame.json
+├── freemocap_sample_data_by_trajectory.csv
+├── freemocap_sample_data_camera_calibration.toml
+├── freemocap_sample_data_frame_name_xyz.npy
+├── output_data
+├── saved_data
+└── synchronized_videos
+```
+
+The contents of the folder described here are for a complete recording. Depending on your processing parameters, or if you encounter an error during processing, some of these files may be missing.
+
+>To get a copy of the sample data to follow along with this guide, you can:
+>1. Open FreeMoCap
+>2. Go to Data in the menu bar
+>3. Select `Download Sample Data`
+>4. Process the sample data by clicking `Process Motion Capture Videos` in the `Process Data` tab
+>5. Navigate to `freemocap_data/recording_sessions/freemocap_sample_data`
+
+
+## Top Level Files
+The top level files in a recording folder, or the files that aren't in a subfolder, are the recommended outputs. These files contain the fully processed data, and formatted for easy use in other programs. FreeMoCap provides this final data in the form of `.npy`, `.csv`, and .`json` files, covering the most common data formats. In addition to the data files, the top level of the recording folder contains output files to help you directly view and use your data. These include a `.blend` file for displaying and animating your data, and a `.ipynb` notebook to guide you through interacting with your data in Python.
+
+### Data Files
+`freemocap_sample_data_frame_name_xyz.npy` is a numpy data array, where the first dimension is the frame number, the second dimension is the keypoint, and the third dimension is the xyz location of the keypoint. 
+
+`freemocap_sample_data_by_trajectory.csv` is comma separated values file, where each column is a trajectory of a keypoint in one dimension, and each row is a frame. The first two columns are `timestamp` and `timestamp_by_camera`, giving the timestamps for each frame if they're available. The rest of the columns are the trajectory of each tracked keypoint, split into a column for each xyz dimension. The column names look like `body_nose_x`, `right_forearm_y`, and `full_body_com_z`.
```

Comment:
What does `timestamp_by_camera` contain? Does it mean that if there are three cameras, this single column contains three timestamps?